content/How to use Lottie Animations on SwiftUI.md

```diff
@@ -0,0 +1,64 @@
+![Main image](images/lottie-cover.png)
+
+# How to use Lottie Animations in SwiftUI
+
+## Introduction
+
+If you're here already, there's a 99.999% chance you know what Lottie is. For those still confused on what this article is about: Lottie is an awesome iOS, Android, and React Native library created by Airbnb that renders After Effects animations in real time, allowing you to use animations as easily as you use static images. You can find more info on their [website](https://airbnb.design/lottie/) and all the technical documentation you want on their [github](https://github.com/airbnb/lottie) page.
+
+
+## Overview
+
+In this tutorial, we’ll create a simple SwiftUI view using `UIViewRepresentable` in order to create and load any Lottie animation into your project.
+
+## Lets get started
+
+### Installation
+
+In order to use Lottie inside your project we need to install the library. For this we have a few different options like Cocapods, Carthage or the one we will use in this tutorial Swift Package Manager.
+
+First of all, on Xcode click on `File` and then select the `Add Package Dependency...` option in Swift Packages. In the input box, enter: `https://github.com/airbnb/lottie-ios.git` (this is the library repository from where Xcode will download the code) and then just hit next and leave the major version thats selected by default to use the more recent release for Lottie.
+
+
+### Hosting Lottie on SwiftUI 
+
+Now that you have Lottie already installed on your project we can start creating the view. For this you'll need to create a new SwiftUI file. Call it `LottieView` and delete everything except the import statement and add the following:
+
+https://gist.github.com/pMalvasio/93a02c5fb8d2f35509d78df8571915cd
+
+So far we're using a `UIViewRepresentable` to allocate an empty `UIView` with frame `.zero`.
+
+
+### Loading the Animation
+
+Next, you'll need to import Lottie on your code, so yo can add `import Lottie` below the SwiftUI import that's already there. On the `makeUIView(...)` method you can now call Lottie and load an animation and add it to the view we already created.
+
+https://gist.github.com/pMalvasio/82ec0abbdcf9b1776a533e673e8f6533
+
+
+### Adding the constraints
+
+To ensure that it takes the full height and width of the container, we'll specify constraints. To do so, add the following before returning the view:
+
+https://gist.github.com/pMalvasio/06844e05841cc42cc26797e9e6d38ff1
```

Comment:
don't you need centering constraints as well?

Reply:
Nope, the Lottie animation will be centered on the hosted view. and you can center or change size from your SwiftUI view where you're using this new component

Reply:
hmmm not sure if i'm following, with UIKit autolayout won't know how to place the view without origin constraints. If no constraints warning are thrown in the console I guess its ok.

Reply:
Don't you have to set the same origin or center for the view?  I'd do it just in case. It might be working because it may be taking 0,0 as the default origin, but I don't know if that will work every time.

Reply:
it seems theres some SwiftUI magic involved, otherwise autolayout would be complaining